packages/frontend/src/components/Explorer/SaveChartButton/index.tsx

```diff
@@ -40,21 +55,40 @@ const SaveChartButton: FC<{ isExplorer?: boolean }> = ({ isExplorer }) => {
 
     return (
         <>
-            <Button
-                disabled={isDisabled}
-                variant={isExplorer ? 'default' : undefined}
-                color={isExplorer ? 'blue' : 'green.7'}
-                size="xs"
-                loading={update.isLoading}
-                leftIcon={
-                    isExplorer ? (
-                        <MantineIcon icon={IconDeviceFloppy} />
-                    ) : undefined
+            <Tooltip
+                label={
+                    'A custom metric ID matches an existing table metric. Rename it to avoid conflicts.'
                 }
-                onClick={handleSaveChart}
+                disabled={!foundCustomMetricWithDuplicateId}
+                withinPortal
+                multiline
+                position={'bottom'}
+                maw={300}
             >
-                {savedChart ? 'Save changes' : 'Save chart'}
-            </Button>
+                <Button
+                    disabled={isDisabled}
+                    variant={isExplorer ? 'default' : undefined}
+                    color={isExplorer ? 'blue' : 'green.7'}
+                    size="xs"
+                    loading={update.isLoading}
+                    leftIcon={
+                        isExplorer ? (
+                            <MantineIcon icon={IconDeviceFloppy} />
+                        ) : undefined
+                    }
+                    {...(isDisabled && {
+                        'data-disabled': true,
+                    })}
+                    sx={{
+                        '&[data-disabled="true"]': {
+                            pointerEvents: 'all',
```

Comment:
Change to show tooltip when button is disabled.

Reply:
I hate we need to need to add a lot of boilerplate just to add a tooltip, it would be nice to have something like `title` , wihin the button component.